magento2-nginx/fpm-7.0/usr/local/share/magento2/install_magento.sh

```diff
@@ -28,9 +28,16 @@ fi
 
 if [ ! -d "vendor" ] || [ ! -f "vendor/autoload.php" ]; then
   as_code_owner "composer config repositories.magento composer https://repo.magento.com/"
-  as_code_owner "composer global config http-basic.repo.magento.com '$MAGENTO_USERNAME' '$MAGENTO_PASSWORD'"
-  as_code_owner "composer global config http-basic.toran.inviqa.com '$TORAN_USERNAME' '$TORAN_PASSWORD'"
-  as_code_owner "composer global config github-oauth.github.com '$GITHUB_TOKEN'"
+
+  if [ -n "$MAGENTO_USERNAME" ]; then
```

Comment:
do we need to check that magento username and password are set? or could password be blank?